app/models/establishment.rb

```diff
@@ -7,6 +7,18 @@ class Establishment < ApplicationRecord
 
   has_many :invitations, dependent: :nullify
 
+  has_many :establishment_users, dependent: :destroy
```

Comment:
J'aime bien voir `_relationship` à la fin du nom de mes tables de relation n to n, mais ça se discute.